be/src/service/staros_worker.h

```diff
@@ -53,6 +53,8 @@ class StarOSWorker : public staros::starlet::Worker {
 
     absl::Status add_shard(const ShardInfo& shard) override;
 
+    absl::Status modify_shard(const ShardId shard, const bool enable_cache) override;
```

Comment:
Question: when will this method been invoked?

Reply:
fe -> starManager: alterShard
starManager -> StarOS_worker: modifyShard

Reply:
The function name of modify_shard() was too generic, make it more precisely capture what the function does.